### DISCUSSION TITLE
Inquiry Regarding liboqs Support for NIST Finalized PQC Algorithms

### DISCUSSION THREAD

**Discussion by Dhanasree234** [open-quantum-safe/liboqs#1903] [General]
With NIST recently finalizing post-quantum cryptography (PQC) algorithms, I would like to inquire whether there are plans for liboqs (specifically regarding IKE/KEM) to release an update that aligns with the finalized list of NIST algorithms.

Your insights on this would be greatly appreciated.

Thank you for your time and support.

**Comment by bhess** [comment]
Hi @Dhanasree234, yes we're working on integrating the finalized NIST standards, likely for a 0.12.0 release. See also the Roadmap discussion #1892 and the issue tracking the liboqs integration of ML-KEM/ML-DSA: https://github.com/open-quantum-safe/liboqs/issues/1891.
